CMakeLists.txt

```diff
@@ -70,6 +73,10 @@ elseif(UNIX)
   endif()
 endif()
 
+if((LOVR_BUILD_SHARED AND NOT ANDROID) OR LOVR_USE_PICO)
+  set(LOVR_USE_MAIN OFF)
+endif()
```

Comment:
Just got another idea for this -- instead of the `LOVR_USE_MAIN` option, why not use `LOVR_BUILD_EXE`?

```cmake
if(NOT ANDROID AND LOVR_BUILD_SHARED)
  set(LOVR_BUILD_EXE OFF)
endif()

# later ...

if(LOVR_BUILD_EXE AND NOT LOVR_USE_PICO)
  list(APPEND LOVR_SRC src/main.c)
else()
  target_compile_definitions(lovr LOVR_OMIT_MAIN) # specifically for win32 WinMain
endif()
```

Reply:
yeah that's nice, less options. implemented.